policies/tagging/policy.json

```diff
@@ -0,0 +1,113 @@
+{
+    "properties": {
+      "displayName": "HMCTS Tagging Policy",
+      "policyType": "Custom",
+      "mode": "Indexed",
+      "parameters": {
+        "excludedResourceTypes": {
+          "defaultValue": [
+            "Microsoft.Network/networkWatchers",
+            "Microsoft.Web/staticSites"
```

Comment:
not sure if we should ignore this, it should probably show up in the audit.

I assume the fix is trivial